resources/assets/js/src/components/pages/Login.tsx

```diff
@@ -1,24 +1,32 @@
 import * as React from "react";
-import {Button, Col, ControlLabel, form, FormControl, FormGroup, Row} from "react-bootstrap";
+import {Button, Col, ControlLabel, Form, FormControl, FormGroup, HelpBlock, Row} from "react-bootstrap";
 import {connect} from "react-redux";
 import { Redirect } from "react-router-dom";
 import * as AlertStyles from "../../config/constants/alertStyles";
+import * as Errors from "../../config/constants/errors";
+import * as validators from "../../config/validators";
 import * as actions from "../../store/actions";
 import Alert from "../alert/Alert";
 import Loading from "../loading/Loading";
 
 const formInputs = [
     {
+        errorMessage: null,
         label: "E-mail",
         name: "email",
         placeholder: "E-mailová adresa",
-        type: "text",
+        type: "email",
+        validationState: null,
+        validations: [validators.required],
```

Comment:
Missing, email validator?

Reply:
E-mail validator is prepared for future use, now it's solved by build-in HTML5 form validation